### DISCUSSION TITLE
Deleted

### DISCUSSION THREAD

**Discussion by Rexadev** [Eloston/ungoogled-chromium#1891] [Ideas]

**Comment by networkException** [comment]
What would be the benefit of this?

**Comment by networkException** [comment]
Afaik it never actually gets to making a DNS query, at least my pihole doesn't get any requests.

**Comment by wchen342** [comment]
Then it is probably a leak somewhere. However I agree the name shall be changed in case of such a leak as a defense in depth measure. `.example`/`.invalid` is part of RFC 2606 which is more standard and I agree it seems a better choice.

**Comment by wchen342** [comment]
About the leak there are several reasons I can think of why @Rexadev saw a leak but @networkException didn't:
- Windows builds are not tested as thoroughly as Linux ones so they may have some unexpected behaviors.
- If it was the woolyss builds, they may have their own modifications, we really have no idea.

**Comment by fulldecent** [comment]
Is this discussion live and should it be linked from mention of suffix at https://github.com/ungoogled-software/ungoogled-chromium/blob/master/docs/design.md#source-file-processors
